src/modules/app/containers/I18nProvider.js

```diff
@@ -24,35 +21,11 @@ export class I18nProvider extends React.Component {
   constructor () {
     super()
 
-    const i18nextResources = I18nProvider.transformResources(localesResources)
-    this.i18n = i18n.createInstance()
-      .use(LanguageDetector)
-      .init({
-        resources: i18nextResources,
-        fallbackLng: FALLBACK_LANGUAGE,
-        load: 'languageOnly',
-        // eslint-disable-next-line no-undef
-        debug: __DEV__
-      })
+    this.i18n = i18n
 
     this.state = {language: FALLBACK_LANGUAGE}
   }
 
-  /**
-   * Transform locale resources to the structure: languageCode -> namespace -> key:value
-   * And not: namespace -> languageCode -> key:value
-   * @param {object} resources
-   * @returns {object} transformed resources suplliable to i18next instance
-   */
-  static transformResources (resources) {
-    return reduce(resources, (accumulator, namespace, namespaceName) => {
-      forEach(namespace, (language, languageCode) => {
-        accumulator[languageCode] = {...accumulator[languageCode], [namespaceName]: language}
-      })
-      return accumulator
-    }, {})
-  }
-
```

Comment:
Fonts don't load for me

Reply:
yes, for me they load sometimes but sometimes not. Could we also discuss/fix this on wednesday?

Reply:
Did you fix this?
If not, you could add it to the i18n task

Reply:
@Schedulaar fixed (by reverting the changes made to the I18nProvider and readding the I18nRedirectPage)